libraries/rush-lib/src/cli/scriptActions/PhasedScriptAction.ts

```diff
@@ -576,13 +591,18 @@ export class PhasedScriptAction extends BaseScriptAction<IPhasedCommandConfig> {
               break;
           }
         }
+        buildTimings = _setBuildTimes(result);
+        dependencyGraph = makeDependencyGraph(result);
       }
 
       this.parser.telemetry.log({
         name: this.actionName,
         durationInSeconds: stopwatch.duration,
         result: success ? 'Succeeded' : 'Failed',
-        extraData
+        extraData,
+        machineInfo,
+        buildTimings,
+        dependencyGraph
```

Comment:
Once we're including this data in the telemetry file, the summary fields above become redundant and we may as well just list out the full dependency graph, with the timing and status for each operation in the graph.